src/ActivityRepository.js

```diff
@@ -85,31 +85,33 @@ class ActivityRepository {
     return this.user.slice(index - 6, index + 1);
   }
 
-  getPositiveStepTrends() {
-    return this.user.reduce((acc, day, index) => {
-      if (index < 2) {
+  getStepTrends(type) {
```

Comment:
A better parameter name here might be `trendDirection` - type is a little vague

Reply:
Thanks for the suggestion, Brittany! I renamed that and made another push.